bundle-workflow/src/perf_test.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import os
+
+import yaml
+
+from git.git_repository import GitRepository
+from manifests.bundle_manifest import BundleManifest
+from system.working_directory import WorkingDirectory
+from test_workflow.perf_test_cluster import PerformanceTestCluster
+from test_workflow.perf_test_suite import PerformanceTestSuite
+
+parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
```

Comment:
Please add comments for how structure of this file should work since it doesn't follow the typical interface/inheritance pattern

Reply:
I have added the comment and explained the required arguments for Perf Test. We will invoke this file from `test.sh`